src/Components/AddComment.jsx

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable react/prop-types */
+import { useState } from "react";
+import postComment from "../../utils/add-comment";
+import getCommentsById from "../../utils/get-comments-by-id";
+
+function AddComment(props) {
+  const { articleId } = props;
+  const [username, setUsername] = useState("");
+  const [comment, setComment] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postComment(articleId, comment, username).then((response) => {
+      if (response) {
+        setUsername("");
+        setComment("");
+        getCommentsById();
+        alert("Comment posted successfully");
```

Comment:
Please avoid using alerts to provide user feedback, for a few reasons
- They're generally found frustrating by users because you have to click on them before you continue
- Some browsers block pop ups so they won't even see them 👀 
If you would like a pop up message use something like Toast - where the pop up happens within the page and doesn't necessarily need input from the user
https://ireade.github.io/Toast.js/